lib/tasks/developer_digest.rake

```diff
@@ -0,0 +1,37 @@
+namespace :developer_digest do
```

Comment:
I followed your lead and omitted tests for the mailer itself. I'm not sure if we want them for these rake tasks. I figured I'd ask before writing them, since after some reading, it looked like writing rake task tests is a bit complicated.

Reply:
I'm fine omitting tests for the mailer, especially because you included the previews.

But I think because this rake task has some real logic we want some sort of tests for it. I recommend pulling out all of the code into a model and testing _that_. Then the rake task can call a single line and we don't have to worry about testing it.